ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v1/container/clientlibs/editor/js/editDialog.js

```diff
@@ -79,6 +94,108 @@
         }
     }
 
-    Utils.initializeEditDialog(EDIT_DIALOG)(handleAsyncSubmissionAndThankYouOption, handleSubmitAction);
+    function loadSubmitActionSubDialogClientLibs(dialog) {
+        var containerSubmitAction = dialog.find(CONTAINER_SUBMITACTION)[0];
+        containerSubmitAction.on("change", function(){
+            loadRestEndPointDialogClientlibs(dialog);
+            loadFDMDialogClientlibs(dialog);
+            loadEmailDialogClientlibs(dialog);
+        });
+    }
+
+    function showDorBindRefSelector (dialog) {
```

Comment:
Please write test cases for this code.

Reply:
We cannot write tests for Site editor since we haven't yet exposed adaptive forms container component. Current tests which are running are only for forms editor, sites editor are skipped.

Reply:
Even the form editor test is less, can you add more as per the functionality ?

Reply:
added tests